packages/admin/resources/views/components/layouts/app/sidebar/group.blade.php

```diff
@@ -18,7 +18,7 @@ class="flex items-center justify-between w-full"
             <x-heroicon-o-chevron-down :class="\Illuminate\Support\Arr::toCssClasses([
                 'w-3 h-3 text-gray-600',
                 'dark:text-gray-300' => config('filament.dark_mode'),
-            ])" x-show="$store.sidebar.groupIsCollapsed(label)" />
+            ])" x-show="$store.sidebar.groupIsCollapsed(label)" x-cloak="" />
```

Comment:
`x-cloak` shouldn't have `""` in this case.
```suggestion
            ])" x-show="$store.sidebar.groupIsCollapsed(label)" x-cloak />
```

Reply:
This was due to it being on a blade component. If it's left as `x-cloak` Blade renders it as `x-cloak="1"` which didn’t match the CSS selector. That’s actually a bug I accidentally introduced in the sidebar PR. 😕

I’ve implemented a [better solution](https://github.com/laravel-filament/filament/pull/1486/commits/c52fc9a70c8a602e6a563de93b5d5499eaaeff79) which fixes that bug.